portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,176 @@
+                    </p>
+                </div>
+            </div>
+
+            <!-- image about twins talk being a twin -->
+            <div class="column"> 
+                <div class="content"> 
+                    <img  src="images/Twins.jpeg" alt="oh no">
+                    <h3>Being a Twin</h3>
+                    <p> 
+                        My twin sister Isabella and I we are best friends but 
+                        also very different. We are polar opposites of each other.
+                        My sister very arty. I'm very math and sciencey. 
+                    </p>
+                </div>
+            </div>
+
+            <!-- image on Miami where i grew up -->
+            <div class="column"> 
+                <div class="content"> 
+                    <img  src="https://blog-www.pods.com/wp-content/uploads/2019/08/MG_6_1_Miami.jpg" alt="oh no" >
+                    <h3>Miami</h3>
+                    <p>
+                        Miami the place were I was raised. A melting pot of 
+                        different Latino cultures. Of course, We have a ton of 
+                        Cubans because you know Miami is known for their 
+                        Cubans, Mis Cubanitos. But, I love being from here and living here.  
+                    </p>
+                </div>
+            </div>
+
+            <!-- image on bad bunny my fav artist  -->
+            <!-- maybe use the video tag and add a bad bunny video?? -->
+            <div class="column"> 
+                <div class="content"> 
+                    <img  src="https://cdn3.pitchfork.com/longform/976/badbunnyheader.jpg" alt="bad bunny" >
+                    <h3>Bad Bunny</h3>
+                    <p>
+                        Bad Bunny my all time favorite music artist.
+                        Bad Bunny is huge in the latino community and I love
+                        his music and videos. Also, I went to his last concert 
+                        in Miami. It was insane, he was soooo good live 10/10
+                        recommend. 
+                    </p>
+                </div>
+            </div>
+
+            <div class="column"> 
+                <div class="content"> 
+                    <img  src="https://static.iris.net.co/semana/upload/images/2018/6/1/569673_1.jpg" alt="Soccer" >
+                    <h3>Soccer(Futbol)</h3>
+                    <p>
+                        I love soccer and have loved it all my life.
+                        Growing up my family and I would watch soccer matches
+                        on the tv a lot. We alway watch parties during the Fifa World Cup
+                        to watch Colombia games. It was a lot of fun. Let's go 
+                        Colombia!
+                    </p>
+                </div>
+            </div>
+
+            <div class="column"> 
+                <div class="content"> 
+                    <img  src="images/shpe.png" alt="Shpe" >
```

Comment:
alt should have "Society of Professional Hispanic Engineers" spelled out probably. Remember that the purpose of the alt is for people who have low vision who are using screen readers, so that they know what is in the image. An acronym probably won't convey that in this case.

Reply:
That very true I will change that right away